postmark-settings.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Override settings
+ *
+ * @return array $settings Settings overridden by constants
+ */
+
+$settings = json_decode( get_option( 'postmark_settings' ), true );
+$settings_from_constants = array(
+    'api_key'        => defined('POSTMARK_API_KEY') ? POSTMARK_API_KEY : null,
+    'stream_name'    => defined('POSTMARK_STREAM_NAME') ? POSTMARK_STREAM_NAME : null,
+    'sender_address' => defined('POSTMARK_SENDER_ADDRESS') ? POSTMARK_SENDER_ADDRESS : null,
+);
+$settings = array_merge( $settings, $settings_from_constants );
```

Comment:
@tflight If I do not specify an overridden Postmark API Token the token set in the plugin UI's **Settings** page is not used, leading to this Postmark API error:

`{"ErrorCode":10,"Message":"No Account or Server API tokens were supplied in the HTTP headers. Please add a header for either X-Postmark-Server-Token or X-Postmark-Account-Token."}`

To reproduce:

1. Install PM plugin version `development`
2. Fill out settings, including Postmark API Token, and save changes.
3. Navigate to plugin UI's 'Send Test Email' tab.
4. Send a test email.

<img width="1665" alt="Screen Shot 2021-07-19 at 9 39 43 AM" src="https://user-images.githubusercontent.com/16660335/126195844-f8cb9062-11f1-45e3-b906-4d44e4cd0227.png">

Can you please check the `array_merge` here? It looks to be setting `POSTMARK_API_KEY` to `null` when it is not overridden.

Reply:
Yes, I had seen and fixed that at one point but missed it in the PR. Sorry I missed that.